packages/eslint-plugin/docs/rules/consistent-type-assertions.md

```diff
@@ -29,7 +29,7 @@ Examples of them include `let x = "hello" as const;` and `let x = <const>"hello"
 
 This option defines the expected assertion style. Valid values for `assertionStyle` are:
 
-- `as` will enforce that you always use `... as foo`.
+- `as` will enforce that you always use `... as foo`. (default)
```

Comment:
🤔 the docs page already has an auto-generated code block _just_ above this line that describes the default options:

![Screenshot of the Options section in https://typescript-eslint.io/rules/consistent-type-assertions including const defaultOptions: Options with assertionStyle: "as"](https://user-images.githubusercontent.com/3335181/213490802-1c2d3a7d-cfac-430b-95cd-4c54622696fa.png)

Did you see that @trajano?

Reply:
Ok cool I'll close this out. Thanks though!